### DISCUSSION TITLE
WebGIS6 Bearbeiten - Auswahlwerkzeug

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#180] [General]
Das Auswahlwerkzeug beim Bearbeiten von erfassten Objekten ignoriert den Darstellungsfilter. 
Als Ergebnis werden immer alle Objekte an der Koordinate ausgegeben, auch wenn vielleicht nix mehr sichtbar ist.

Auch `<add key="tolerance-for-polygon-layers" value="0" />` greift beim Auswahlwerkzeug im Bereich _Bearbeiten_ nicht.

**Comment by gis-eni** [comment]
Jedes Werkzeug hat in der ``api.config`` seinen eigenen Konfigurationsblock. Auf die Werte unter ``<section name="tool-identify">`` kann das Editwerkzeug nicht zugreifen. 

Für das Editwerkzeug muss die Werte einfach noch einmal unter dem ``tool-editing`` Block eintragen:

``` xml
	<section name="tool-editing">
	         <!-- ..... -->

		<add key="tolerance" value="15" /> <!-- Standardwert --> <!-- pixel -->
		<add key="tolerance-for-point-layers" value="15" /> <!-- optional -->
		<add key="tolerance-for-line-layers" value="15" />  <!-- optional -->
		<add key="tolerance-for-polygon-layers" value="0" /> <!-- optional -->
	</section>
``` 

*) Für WebGIS 6

**Reply by gis-eni** [reply]
Auch das Problem mit den Darstellungsfilter ist mit den nächsten Build behoben
